Project_Proposal.md

```diff
@@ -0,0 +1,30 @@
+### Name of the game: 
+# Brain on blue
+
+### Short description of gameplay: 
+Side scrolling  adventure game of a brain named BOB.
+Bob is searching for a head to inhabit but must travel
+overcome various obstacles to eventually find the head.
+
+### Win condition:
+BOB is able to navigate past all obsolesces and find a head
+
+### Lose condition:
+Obstacles cause enough damage that BOB dies
+
+### How the player interacts with the game:
+the interface will be via the keyboard using the "up", "Down",
+"left", and right arrow keys.
+
+### List of game features:
+* side scrolling adventure
+* interesting obstacles to navigate with left right and jump
```

Comment:
Might be good to elaborate on obstacle types. Are you just wanting hazards that damage/kill the player or do you want moving platforms, conveyor belts, or something else?

Having a list of specific obstacle types you want will help you figure out what is actually feasible for you to implement.